### PR TITLE
VAOS updates for VAOS service schema changes

### PR DESCRIPTION
**Pull request by jbalboni**
## Description

Changes are:
- Updates cancellationReason to cancelationReason, in transformer and UI code
- Switches to practitioners from practitionerIds when submitting
- Sends additional info in comment instead of reason field
- Updates patient metadata endpoint url and transformer
- Added new type of care ids for audiology types

## Original issue(s)
department-of-veterans-affairs/va.gov-team#30605

## Testing done
Unit testing

## Acceptance criteria
- [ ] Build passes

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/services/mocks/index.js [hunk 1]
```diff
@@ -271,6 +271,44 @@ const responses = {
...
+
+    if (
+      isDirect &&
+      (!req.query.facility_id.startsWith('984') ||
+        req.query.clinical_service_id === 'primaryCare')
+    ) {
+      ineligibilityReasons.push({
+        coding: [
+          {
+            code: 'ineligible-history',
```
Comment (jbalboni) [inline src/applications/vaos/services/mocks/index.js:286]: I don't know what these codes are yet, so will need to update after that happens

File: src/applications/vaos/services/appointment/transformers.v2.js [hunk 2]
```diff
@@ -164,7 +164,7 @@ export function transformVAOSAppointment(appt) {
     resourceType: 'Appointment',
     id: appt.id,
     status: appt.status,
-    cancellationReason: appt.cancellationReason,
+    cancelationReason: appt.cancelationReason,
```
Comment (vbahinwillit) [inline src/applications/vaos/services/appointment/transformers.v2.js:167]: Always wondered about one 'L' or two 'LL' for cancelationReason. Is it consistently one 'L' now?

**Review by vbahinwillit — APPROVED** [review]

File: src/applications/vaos/services/appointment/transformers.v2.js (hunk 2 above)
Comment (jbalboni) [reply src/applications/vaos/services/appointment/transformers.v2.js:167]: Yeah, it should be just one l everywhere now. There are still instances of just "cancellation" in the code, but I think we can clean those up, too.

**Review by vbahinwillit — APPROVED** [review]

File: src/applications/vaos/tests/mocks/fetch.js [hunk 3]
```diff
@@ -45,7 +45,7 @@ export function mockEligibilityFetchesByVersion({
       directReasons.push({
         coding: [
           {
-            code: 'ineligible-history',
+            code: 'patient-history-insufficient',
```
Comment (vbahinwillit) [inline src/applications/vaos/tests/mocks/fetch.js:48]: Minor but 'VAOS_SERVICE_PATIENT_HISTORY' constant could be used.

**Review by vbahinwillit — APPROVED** [review]

**Pull request merged**